Meta/run.py

```diff
@@ -407,11 +407,10 @@ def set_up_virtualization_support(config: Configuration):
     # even if we couldn't detect it otherwise; this is intended behavior.
     if provided_virtualization_enable is not None:
         config.virtualization_support = provided_virtualization_enable == "1"
-    else:
-        if (config.kvm_usable and config.architecture == Arch.x86_64 and os.uname().machine == Arch.x86_64.value) or (
-            config.qemu_kind == QEMUKind.NativeWindows and config.architecture == Arch.x86_64
-        ):
-            config.virtualization_support = True
+    elif config.architecture == Arch.x86_64 and os.uname().machine == Arch.x86_64.value:
+        # FIXME: Can RISC-V use hardware acceleration?
+        config.virtualization_support = (config.qemu_kind == QEMUKind.NativeWindows or kvm_usable()
+                                         or os.uname().sysname == "Darwin")
```

Comment:
are you sure this doesn't cause issues for x86_64 serenity on apple silicon?

I suggested in https://github.com/SerenityOS/serenity/pull/23592#discussion_r1525615277 that we could check for hvf in `-accel` first.

Reply:
We throw out HVF below if it's not actually supported:

```python3
        # Check if HVF is actually available if we're on MacOS
        if config.qemu_kind == QEMUKind.MacOS and (b"hvf" not in available_accelerators):
config.virtualization_support = False
```

Reply:
but not sure, I only have my Intel Macbook with me to test...

Reply:
ah sure. I don't like setting a flag only to reset it later, but I guess this is consistent... 

One the only I'd super want then is to change the uname check for `QEMUKind.MacOS` (why is that MacOS instead of macOS?)

Reply:
Did you mean like this?